configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5-cpu.env

```diff
@@ -0,0 +1,15 @@
+GST_PIPELINE_LAUNCH=/home/pipeline-server/framework-pipelines/yolov5_pipeline/yolov5s.sh
+OCR_DEVICE=
+LOG_LEVEL=ERROR
+GST_DEBUG=0
+LOW_POWER=
+CPU_ONLY=
+AUTO_SCALE_FLEX_140=
+DECODE=decodebin
+CPU_SW_DECODER=force-sw-decoders=1
+DEVICE=CPU
+PRE_PROCESS=
+AGGREGATE=gvametaaggregate name=aggregate !
```

Comment:
the value of the env variable needs to be double quoted since there is space and equal sign in between.